solr/core/src/java/org/apache/solr/handler/admin/LukeRequestHandler.java

```diff
@@ -105,6 +106,11 @@
 
   static final int HIST_ARRAY_SIZE = 33;
 
+  @Override
+  public Name getPermissionName(AuthorizationContext request) {
+    return Name.CONFIG_READ_PERM;
```

Comment:
It also exposes schema which is configuration. But I guess data is more sensitive, so better choose READ for this. Then users can setup custom permissions that require diferent roles to read data from different collections, and this will obey.

Reply:
Done.